blogposts/2022/release-3.38.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Sourcegraph 3.38 release'
+publishDate: 2022-03-21T10:00-07:00
+description: 'Sourcegraph 3.38 introduces improved Notebooks, faster Code Intelligence for large repos, dependencies search, and custom file syntax highlighting.'
+tags: [blog, release]
+slug: 'release/3.38'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+changelogItems:
+  - description: Code Insights will now periodically clean up data series that are not in use. There is a 1 hour grace period where the series can be reattached to a view, after which all of the time series data and metadata will be deleted.
+    url: https://github.com/sourcegraph/sourcegraph/pull/32094
+    category: Code Insights
+  - description: 'Code Insights critical telemetry total count now only includes insights that are not frozen (limited by trial mode restrictions).'
```

Comment:
```suggestion
  - description: 'Code Insights critical telemetry total count now only includes insights that are not frozen (i.e., limited by trial mode restrictions).'
```

Reply:
@Joelkw - what does this mean for customers - what is the benefit?

Reply:
@abschu there's not benefit for customers, this got picked up in the release changelog script and I recommend we cut this out of the highlights section. The benefit to us is we can accurately check that our licensing restrictions are working. (The "benefit" to customers is this prevents longer term effects like risking too many gating features or gating bugs, and we can be a bit laxer with access controls technically.)